### DISCUSSION TITLE
Как при открытии цикла отображать конкретное поле?

### DISCUSSION THREAD

**Discussion by OAndrey1985** [totumonline/totum-ru-issues-and-discussions#536] [Как это сделать?] [answered]
При открытии цикла отображается ID поля цикла, что логично, а можно ли вместо ID отображать например артикул или ФИО, есть такая опция, имеется ввиду слева в дереве.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Это параметр в таблице циклов - Основное поле. Можно склеить его из разных и сделать скрытым, если хотите составную инф туда показывать

**Comment by OAndrey1985** [comment]
Супер спасибо, то что надо
